downstream/modules/eda/proc-eda-delete-credential.adoc

```diff
@@ -2,12 +2,19 @@
 
 = Deleting a credential
 
+You can delete credentials if they are no longer needed for your organization.
+
 .Procedure
 
 . Delete the credential by using one of these methods:
 * From the *Credentials* list view, click the btn:[More Actions] icon *{MoreActionsIcon}* next to the desired credential and click btn:[Delete credential].
 * From the *Credentials* list view, select the name of the credential, click the btn:[More Actions] icon *{MoreActionsIcon}* next to btn:[Edit credential], and click btn:[Delete credential].
 . In the pop-up window, select *Yes, I confirm that I want to delete this credential*.
++
+[NOTE]
+====
+If your credential is still in use by other resources in your organization, a warning message is displayed letting you know that the credential cannot be deleted.
```

Comment:
@jself-sudoku If a Credential is being used in an Event Stream you won't be able to delete it, till the Event Stream is deleted or attached to a different Credential. It's highly recommended to not delete any Credential that is use, since that can lead to broken Activations.